modules/ossm-rn-known-issues.adoc

```diff
@@ -34,11 +34,26 @@ endif::openshift-rosa[]
 
 These are the known issues in {SMProductName}:
 
+* https://issues.redhat.com/browse/OSSM-3890[OSSM-3890] In multitenant mesh deployment, all Gateway API CRDs must be present. Attempting to use the Gateway API in multitenant mesh deployment generates an error message similar to the following:
++
+`2023-05-02T15:20:42.541034Z	error	watch error in cluster Kubernetes: failed to list *v1alpha2.TLSRoute: the server could not find the requested resource (get tlsroutes.gateway.networking.k8s.io)
+2023-05-02T15:20:42.616450Z	info	kube	controller "gateway.networking.k8s.io/v1alpha2/TCPRoute" is syncing...`
++
+For Gateway API support to work in a multitenant mesh deployment, all Gateway API Custom Resource Definition (CRD) files must be present in the cluster. 
++
+In a multitenant mesh deployment, CRD scan is disabled, so Istio has no way to discover which CRDs are present in a cluster. As a result, Istio attempts to watch CRDs, but generates errors because no CRDs are present.
```

Comment:
*attempts to watch all supported Gateway API CRDs, but generates errors because some of those CRDs are not present*